corehq/apps/receiverwrapper/views.py

```diff
@@ -405,23 +346,56 @@ def _secure_post_api_key(request, domain, app_id=None):
 @set_request_duration_reporting_threshold(60)
```

Comment:
I notice this decorator is on both the parent and the child views.  Is it perhaps that one of the decorators doesn't call `functools.wraps` and so the wrapped fn doesn't have the proper attribute set?